client/src/components/CharacterItem.js

```diff
@@ -12,7 +12,7 @@ const CharacterItem = ({ character }) => {
       <Link to={`/character/${id}`}>
         <Card.Content>
           <Image floated='left' size='mini' src={avatar} />
-          <Card.Header>{name}</Card.Header>
+          <Card.Header textAlign='center'>{name}</Card.Header>
         </Card.Content>
       </Link>
     </Card>
```

Comment:
These character cards could be justified better, e.g:

```jsx
const CardContent = styled(Card.Content)`
  display: flex;
  flex-direction: row;
  align-items: center;
`;

const CardHeader = styled(Card.Header)`
  margin-left: 10px;
`;

const CharacterItem = ({ character }) => {
  const { name, id } = character;

  return (
    <Card>
      <Link to={`/character/${id}`}>
        <CardContent>
          <Image size="mini" src={avatar} />
          <CardHeader>{name}</CardHeader>
        </CardContent>
      </Link>
    </Card>
  );
};
```

![image](https://user-images.githubusercontent.com/2788771/80955303-588c7b80-8dff-11ea-916d-b44e3915b125.png)